frontend/amundsen_application/static/js/interfaces/TableMetadata.ts

```diff
@@ -102,6 +103,7 @@ export interface TableMetadata {
   name: string;
   description: string;
   table_writer: TableWriter;
```

Comment:
would it be difficult to change the name from TableWriter to App/TableApp for the benefit of future maintainers.